src/components/Book.js

```diff
@@ -2,22 +2,31 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 const Book = (props) => {
```

Comment:
- [ ] [OPTIONAL] You can destruct the props in the declaration line of the component in order to save lines of code:
```javaScript
const Book = ({book, remove}) => {
```